### PR TITLE
[Reviewer: Graeme] Add mock for the infinite table

### PR DESCRIPTION
**Pull request by eleanor-merry**
Add mock for the infinite table

### CODE REVIEW

File: test_utils/mock_infinite_table.cpp [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ * software and licensed under the OpenSSL Licenses.
+ * "OpenSSL Licenses" means the OpenSSL License and Original SSLeay License
+ * under which the OpenSSL Project distributes the OpenSSL toolkit software,
+ * as those licenses appear in the file LICENSE-OPENSSL.
+ */
+
+#include "mock_infinite_table.h"
+
+MockInfiniteTable::MockInfiniteTable() {}
+MockInfiniteTable::~MockInfiniteTable() {}
```
Comment (graemerobertson) [inline test_utils/mock_infinite_table.cpp]: What's the point of this file? Can't we just put these in the header file and delete this file? Is that bad practice or something? I thought that's what we normally do for mocks.

**Comment by graemerobertson** [comment]
Reviewed

File: test_utils/mock_infinite_table.cpp (hunk 1 above)
Comment (bossmc) [inline test_utils/mock_infinite_table.cpp]: https://github.com/Metaswitch/homestead/pull/150

Comment (graemerobertson) [inline test_utils/mock_infinite_table.cpp]: Fair enough. Keep up the good work Ellie!

**Pull request merged**